objects/Game.js

```diff
@@ -501,5 +501,16 @@ module.exports = {
         })
         
     },
+    async updateLeaderboard(matchID, receivedMessage, players){
+        let winner = true
+        players.forEach(player => {
+            if (winner === true){
```

Comment:
I think this will always be set to true according to the logic above.  You set it to true, then evaluate if it's true 2 lines down.

Reply:
Yea, I see it now.  Thanks.